CHANGELOG.md

```diff
@@ -27,6 +33,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [3.82.0] - 2020-11-10
 ### Added
 - `"showAppliedFiltersOverview"` prop to the `SearchResult`.
+### Added
```

Comment:
Ops, there are two entries in the changelog

Reply:
Sorry about that! Fixed!